### PR TITLE
Milestone 5: Delete

### PR DESCRIPTION
**Pull request by DeannaMix**
## Description
Users can now delete rides, staff, vendors, dinos, equipment.

## Related Issue
#66 #16 #15 #27 #24 

## Motivation and Context
Users who were authorized wanted to be able to delete the different modules.

## How Can This Be Tested?
[deploy](https://jurassic-world-eb567.web.app/)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by brownabs** [comment]
please include site link

File: src/javascripts/components/cards/staffCards.js [hunk 1]
```diff
@@ -31,6 +39,7 @@ const staffCardBuilder = () => {
     .getStaff()
     .then((response) => {
       response.forEach((item) => {
+        console.warn(item);
```
Comment (brownabs) [inline src/javascripts/components/cards/staffCards.js]: remove console.warns from code

**Comment by drteresavasquez** [comment]
Don't see the option to delete on dinos and equipment in the UI

File: src/javascripts/components/cards/dinoCards.js [hunk 2]
```diff
@@ -9,9 +9,16 @@ const authedDinoCardView = (dinoObject) => {
...
           <h3 class="card-text card-header">${dinoObject.name}</h3>
         </div>
-        <button type="button" id="${dinoObject.dinoId}" class="btn btn-info update-dino">Edit</button>
+        <button type="button" id="${dinoObject.dinoId}" class="btn btn-info update-dino card-btns"><i class="fas fa-pen"></i></button>
+        <button type="button" id="${dinoObject.dinoId}" class="btn btn-info delete-dino card-btns"><i class="fas fa-trash-alt"></i></button>
       </div>
     </div>`;
+
+  $('body').on('click', 'button.delete-dino', (e) => {
+    $(`.card#${e.currentTarget.id}`).remove();
```
Comment (brownabs) [inline src/javascripts/components/cards/dinoCards.js]: in staying consistent with your other on click events, maybe consider also storing e.currentTarget.id into firebaseKey variable

File: src/javascripts/components/cards/equipmentCards.js [hunk 3]
```diff
@@ -3,19 +3,25 @@ import 'firebase/auth';
...
         <div>
           <h3 class="card-text card-header">${equipmentObject.name}</h3>
         </div>
-        <button type="button" id="${equipmentObject.equipmentId}" class="btn btn-info update-equipment">Edit</button>
+        <button type="button" id="${equipmentObject.equipmentId}" class="btn btn-info update-equipment card-btns"><i class="fas fa-pen"></i></button>
+        <button type="button" id="${equipmentObject.equipmentId}" class="btn btn-info delete-equipment card-btns"><i class="fas fa-trash-alt"></i></button>
       </div>
     </div>`;
+  $('body').on('click', 'button.delete-equipment', (e) => {
+    $(`.card#${e.currentTarget.id}`).remove();
```
Comment (brownabs) [inline src/javascripts/components/cards/equipmentCards.js]: in staying consistent with your other on click events, maybe consider also storing e.currentTarget.id into firebaseKey variable

**Review by brownabs — CHANGES_REQUESTED** [review]

**Review by drteresavasquez — APPROVED** [review]
Looks good!

File: src/javascripts/components/cards/dinoCards.js (hunk 2 above)
Comment (danapham) [reply src/javascripts/components/cards/dinoCards.js]: done!

File: src/javascripts/components/cards/equipmentCards.js (hunk 3 above)
Comment (danapham) [reply src/javascripts/components/cards/equipmentCards.js]: done!

File: src/javascripts/components/cards/staffCards.js (hunk 1 above)
Comment (danapham) [reply src/javascripts/components/cards/staffCards.js]: done!

**Review by brownabs — APPROVED** [review]

File: src/javascripts/components/cards/rideCards.js [hunk 4]
```diff
@@ -7,19 +7,28 @@ const rideCardMaker = (rideObject) => {
...
     <h3 class="card-header">${rideObject.name}</h3>
-    <button type="button" class="btn btn-light update-ride" id="${rideObject.rideId}">Edit</button>
-    <button type="button" class="btn btn-light" id="${rideObject.rideId}">Delete</button>
+    <button type="button" class="btn btn-light update-ride card-btns" id="${rideObject.rideId}"><i class="fas fa-pen"></i></button>
+    <button type="button" class="btn btn-light delete-rides card-btns" id="${rideObject.rideId}"><i class="fas fa-trash-alt"></i></button>
   </div>
 </div>`;
 
+  $('body').on('click', '.delete-rides', (e) => {
+    console.warn('click', e);
```
Comment (brownabs) [inline src/javascripts/components/cards/rideCards.js]: this is a new console warn?

**Pull request merged**